handlers/session_sync_handler.go

```diff
@@ -54,13 +53,14 @@ func (h *Handler) SessionSyncHandler(w http.ResponseWriter, req *http.Request, p
 	} else {
 		err = h.checkIfK8SConfigExistsOrElseLoadFromDiskOrK8S(req, user, prefObj, provider)
 		if err != nil {
-			logrus.Errorf("unable to initialize kubernetes config")
+			h.log.Error(ErrInvalidK8SConfig)
 		}
 	}
 
 	err = json.NewEncoder(w).Encode(prefObj)
 	if err != nil {
-		logrus.Errorf("error marshaling user config data: %v", err)
+		obj := "user config data"
+		h.log.Error(ErrMarshal(err, obj))
 		http.Error(w, "unable to process the request", http.StatusInternalServerError)
```

Comment:
Hey @Chadha93 I think I missed this so please consider changing the http.Error to `http.Error(w, ErrMarshal(err, obj).Error(), http.StatusInternalServerError)`

Reply:
@DelusionalOptimist @sayantan1413 should I make changes to this only ( if the windows part is not conflicting).